exporters/tests/test_in_memory_metric.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
```

Comment:
all files have this shebang, but they should only have the copyright information, right?

Reply:
yeah, copy paste. All files except app.py should have this line removed.